docs/specification/README.md

```diff
@@ -205,7 +193,7 @@ The attachment index is an index to named attachments within the file. One recor
 
 #### Statistics (op=0x0A)
 
-The statistics record contains statistics about the recorded data. It is the last record in the file before the footer.
+The statistics record contains statistics about the recorded data. It is the last record in the file before the footer. The record must be preceded in the index data section by Channel Info records for any channels referenced in the `channel_message_counts` field. If this is undesirable but some statistics are still desired, the field may be set to a zero-length map. The statistics record is optional.
```

Comment:
It's not strictly required that statistics must be preceded by Channel Info. Yes statistics might reference channel ids but doesn't really care where they are found in the file?

Reply:
if the channel infos don't precede the statistics record, there's no value in setting the field nonempty IMO, since you would need to traverse the entire file to get the channel infos and might as well recompute the counts then. So I think better to require it if not having it wouldn't make sense.